docs/README.md

```diff
@@ -404,6 +412,27 @@ Find all events on the functions namespace
 kubectl get events --sort-by=.metadata.creationTimestamp -n openfaas-fn
 ```
 
+##### Troubleshoot Network Policies
+Network Policies are working properly when OpenFaaS is [deployed with Ingress Controller](https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/README.md#deploy-with-an-ingresscontroller).
```

Comment:
>Network Policies are working properly when OpenFaaS is

The NetworkPolicy configuration is designed to work with a Kubernetes IngressController. If you are using a NodePort or LoadBalancer follow the instructions below.